### ISSUE TITLE
How to make GT ground truth map suitable for UCF trained model?

### ISSUE THREAD

**Issue by li-bowen-official** [Pongpisit-Thanasutives/Variations-of-SFANet-for-Crowd-Counting#35]
Hello!

I have a ground truth for images in the form of coordinate points. Is there a way to turn this into a ground truth density map that I can use to compare to the MSFA UCF predicted density map and also visualise? So that I can perhaps make a confusion matrix.

Also, for the bayesian processed SH datasets, the .npy files, I get that the first 2 columns are the width and height coordinates, but what does the 3rd column represent? Some sort of distance or radius? Is this something that can be used to create the ground truth map, or is this for something else?

Thank you for your constant updates and help!

**Comment by Pongpisit-Thanasutives** [comment]
Thank you for the questions.
- For typical density map creation based on a Gaussian kernel, please take a look at this [density_map.py](https://github.com/pxq0312/SFANet-crowd-counting/blob/master/density_map.py).
- If I am not mistaken, the distance information generated by [bayesian_preprocess_sh.py](https://github.com/Pongpisit-Thanasutives/Variations-of-SFANet-for-Crowd-Counting/blob/master/bayesian_preprocess_sh.py) is for calculating  [posterior label probability](https://github.com/Pongpisit-Thanasutives/Variations-of-SFANet-for-Crowd-Counting/blob/master/losses/post_prob.py), which derives [the Bayesian loss function](https://github.com/Pongpisit-Thanasutives/Variations-of-SFANet-for-Crowd-Counting/blob/master/losses/bay_loss.py). More detailed explanation was originally given in [this paper](https://openaccess.thecvf.com/content_ICCV_2019/papers/Ma_Bayesian_Loss_for_Crowd_Count_Estimation_With_Point_Supervision_ICCV_2019_paper.pdf).

Hope this helps!

**Comment by fatbringer** [comment]
Hello! I have a related question. 

I saw that in density_map.py, the Gaussian kernel is given a sigma of 5. Do you know what would be the rationale for that? 
In the paper you shared they mentioned that a fixed value isn't as good.
